plugins/TagFix_MultipleTag.py

```diff
@@ -57,7 +57,7 @@ def init(self, logger):
 '''Missing `maxheight=*` or `maxheight:*` for a tunnel or a way under a
 bridge.'''))
         self.errors[21101] = self.def_class(item = 2110, level = 2, tags = ['tag'],
-            title = T_('Name present but missing main tag'),
+            title = T_('Unknown named object'),
```

Comment:
Not clear enough.

Reply:
```suggestion
            title = T_('Untagged named object'),
```

?

Reply:
Alternatively:

```suggestion
            title = T_('Name without feature'),
```

Reply:
"Untagged named object" look good for me.